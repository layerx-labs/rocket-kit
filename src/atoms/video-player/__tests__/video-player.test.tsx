import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import 'jest-styled-components';
import { VideoPlayer } from '../../..';

describe('VideoPlayer', () => {
  it('renders', () => {
    const { asFragment } = render(
      <VideoPlayer video="https://www.youtube.com/watch?v=E71ffwc13y4" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
