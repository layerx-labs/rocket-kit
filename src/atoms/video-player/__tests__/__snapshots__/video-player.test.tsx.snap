// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders 1`] = `
<DocumentFragment>
  .c0 {
  position: relative;
  padding-top: 56.25%;
}

<div
    class="c0"
  >
    <div
      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
    />
  </div>
</DocumentFragment>
`;
